modules/drivers/usb_cam/AUTHORS.md

```diff
@@ -0,0 +1,9 @@
+Original Authors
+----------------
+
+ * [Benjamin Pitzer] (benjamin.pitzer@bosch.com)
```

Comment:
We don't include third party code into repo.
How about packing it to a library? Lei Wang could help on this.